File: JavaScript_Courses/JavaScript_Basics/Preparation/03_Conditional_Statements_Advanced_Lab/07_Working_Hours.js
function workingHours(input) {
    let hour = Number(input[0]);
    let day = input[1];

    if ((day == "Monday" || day == "Tuesday" || day == "Wednesday" || day == "Thursday"
        || day == "Friday" || day == "Saturday") && (hour >= 10 && hour <= 18)) {
            console.log("open");
        } else {
            console.log("closed");
        }
}

// workingHours(["11", "Monday"]);

File: Python_Courses/Python_Basics/Preparation/03_Conditional_Statements_Advanced_Lab/07_Working_Hours.py
working_days = [
    "Monday",
    "Tuesday",
    "Wednesday",
    "Thursday",
    "Friday",
    "Saturday",
]

non_working_days = [
    "Sunday",
]

hour = int(input())
weekday = input()

if weekday in working_days and 10 <= hour <= 18:
    print("open")
else:
    print("closed")
